main.py

```diff
@@ -473,15 +518,12 @@ def update_outs(
     for j in trange(len(inps_tensor), desc="calc outs after quantization", leave=False):
         outs_batch = layer(inps_tensor[j].to(device).unsqueeze(0), **forward_args)[0]
         if compute_mse:
+            batch_size = outs_batch.shape[0]
             outs_batch_loss = (
-                (outs_batch - outs_tensor[j].to(device))
-                .float()
-                .square()
-                .view(outs_batch.shape[0], -1)
-                .mean(dim=1)
-                .sqrt()
+                (outs_batch - outs_tensor[j].to(device)).float().square().view(batch_size, -1).mean(dim=-1)
```

Comment:
Minor: the new formula looks similar to the previous one, but subtly different: it divides by squared norm instead of variance/std (without subtracting mean). 
This looks like a minor change but let's double check: is this intentional?

Reply:
I think this makes more sense. It is signal to noise ratio.